src/Extensions/DependencyInjection/FunctionsStartup.cs

```diff
@@ -11,21 +10,43 @@ namespace Microsoft.Azure.Functions.Extensions.DependencyInjection
     /// Interface defining a startup configuration action that should be performed
     /// as part of the Functions runtime startup.
     /// </summary>
-    public abstract class FunctionsStartup : IWebJobsStartup
+    public abstract class FunctionsStartup : IWebJobsStartup2, IWebJobsConfigurationStartup
     {
```

Comment:
Trying to remember what we decided with the `IWebJobsStartup2`, I thought we had discussed removing it, but it feels like ages ago.

Reply:
It was required -- we needed a new property to hold the Configure() call with a Context.